api/v1beta2/foundationdb_env_variables.go

```diff
@@ -21,6 +21,75 @@
 package v1beta2
 
 const (
-	// EnvNamePublicIP defines the FDB_PUBLIC_IP environment variable name.
+	// EnvNameDNSName specifies the DNS locality (identifies the pod when using DNS in lieu of IP)
+	EnvNameDNSName = "FDB_DNS_NAME"
+
+	// EnvNameMachineID specifies the Machine ID locality. Also defines fault domain along with EnvNameZoneID
+	EnvNameMachineID = "FDB_MACHINE_ID"
+
+	// EnvNameZoneID specifies Zone ID locality. Also defines fault domain along with EnvNameMachineID.
+	// The current default value of EnvNameZoneID is the hostname
+	EnvNameZoneID = "FDB_ZONE_ID"
+
+	// EnvNameClusterFile specifies the path to the cluster file / file containing the connection string.
+	EnvNameClusterFile = "FDB_CLUSTER_FILE"
+
+	// EnvNameBinaryDir specifies the path of the FDB binary's directory
+	EnvNameBinaryDir = "BINARY_DIR"
+
+	// EnvNameAdditionalEnvFile if specified for the `foundationdb-kubernetes-sidecar` and `foundationdb-kubernetes-init`
+	// containers, its content will be sourced before any container command runs, and you can override or define there
+	// any other environment variable; this can be used for example to inject environment variables using a shared volume
+	EnvNameAdditionalEnvFile = "ADDITIONAL_ENV_FILE"
+
+	// EnvNameTLSCaFile specifies the path to the certificate authority file for TLS connections
+	EnvNameTLSCaFile = "FDB_TLS_CA_FILE"
+
+	// EnvNameTLSCert specifies the path to the certificate file for TLS connections
+	EnvNameTLSCert = "FDB_TLS_CERTIFICATE_FILE"
+
+	// EnvNameTLSKeyFile specifies the path to the key file for TLS connections
+	EnvNameTLSKeyFile = "FDB_TLS_KEY_FILE"
+
+	// EnvNameTLSVerifyPeers specifies the peer verification rules for incoming TLS connections to the split-image sidecar.
+	// See https://apple.github.io/foundationdb/tls.html#peer-verification for the format
+	EnvNameTLSVerifyPeers = "FDB_TLS_VERIFY_PEERS"
+
+	// EnvNameFDBNetworkSunsetThing specifies whether to ignore the failure to initialize some of the external clients
+	// TODO FDB 7.3 adds a check for loading external client library, which doesn't work with 6.3.
+	//  Consider remove this option once 6.3 is no longer being used.
+	EnvNameFDBNetworkSunsetThing = "FDB_NETWORK_OPTION_IGNORE_EXTERNAL_CLIENT_FAILURES"
```

Comment:
This is only used in the testing fixtures right? So I would prefer not to expose this over the `fdbv1beta2` API.

Reply:
yup, gonna move that to e2e